roles/common/tasks/main.yml

```diff
@@ -7,6 +7,8 @@
     - empty.yml
   tags:
     - vars
+    # We need these vars for the entitlements tag to work
+    - entitlements
```

Comment:
Yeah, I quit using the 'vars' tag because I always forgot to include it when using --tags for tasks that required vars to be loaded.  Instead, I've started using ``always`` so I don't have to remember or specially tag include_vars blocks.

Just a suggestion, I find it easier this way.

http://docs.ansible.com/playbooks_tags.html#special-tags